src/js/components/awardv2/shared/InfoTooltipContent.jsx

```diff
@@ -627,3 +627,13 @@ export const ContractAwardAmountsInfo = (
     </div>
 );
 
+export const CFDAOverviewInfo = (
+    <div>
+        <div className="info-tooltip__title">
+            CFDA Program / Assistance Listing Information
+        </div>
+        <div className="info-tooltip__text">
+            <p>Tooltip text not yet approved.</p>
```

Comment:
Since we are releasing this this coming sprint. Can we change this text to `Coming Soon` or some other text that might be appropriate for a user to read.

Reply:
design and testing approved this text for now.